### ISSUE TITLE
After logging in, or if logging in while _already_ logged in, i'm shown a confusing page

### ISSUE THREAD

**Issue by sallyom** [redhat-developer/podman-desktop-redhat-account-ext#57]
I don't see an issue for this already, anyone else seeing this? 

<img width="1411" alt="Screenshot 2024-02-16 at 10 23 23 AM" src="https://github.com/redhat-developer/podman-desktop-redhat-account-ext/assets/11166065/9bf5785e-5fbb-4da6-9125-872f4ca704b4">

**Comment by dgolovin** [comment]
It is similar to https://github.com/redhat-developer/podman-desktop-redhat-account-ext/issues/49, when I see not content for the page for some reason.

**Comment by dgolovin** [comment]
The reason could be that current implementation of serving the page is very slow and by the time the page is sent to browser localhost sever has already got the callback from sso, got the token and stopped.

**Comment by stefwalter** [comment]
I see this too, and the following log output:

```
11:31:09 AM [main] [redhat-authentication] [Error: ENOENT: no such file or directory, open '/data/.local/share/containers/podman-desktop/plugins/quayioredhatdeveloperpodmandesktopredhataccountext/www/success.html'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/data/.local/share/containers/podman-desktop/plugins/quayioredhatdeveloperpodmandesktopredhataccountext/www/success.html'
}
11:31:10 AM [main] [redhat-authentication] Subscription manager execution returned exit code: 1
11:31:14 AM [main] [Info  - 10:31:14.876] Token available from cache, expires in 894998 milliseconds

11:31:16 AM [main] [redhat-authentication] Registry already registered, skipping registration

11:31:32 AM [main] [redhat-authentication] Subscription manager subscription activation check returned exit code: 1
11:31:32 AM [main] [Info  - 10:31:32.782] Token available from cache, expires in 877092 milliseconds

11:31:34 AM [main] [redhat-authentication] Subscription manager registration returned exit code: 1
```

**Comment by stefwalter** [comment]
I can confirm that with the fix in #58 I no longer see this bug.
